src/prefect/settings.py

```diff
@@ -408,1388 +209,1015 @@ def warn_on_misconfigured_api_url(values):
-# __base__ to the following class ensures that mkdocstrings properly generates
-# reference documentation. It does not support module-level variables, even if they are
-# an object which has __doc__ set.
+    runner_process_limit: int = Field(
+        default=5,
+        description="Maximum number of processes a runner will execute in parallel.",
+    )
 
+    runner_poll_frequency: int = Field(
+        default=10,
+        description="Number of seconds a runner should wait between queries for scheduled work.",
+    )
 
-@add_cloudpickle_reduction
-class Settings(SettingsFieldsMixin):
-    """
-    Contains validated Prefect settings.
-
-    Settings should be accessed using the relevant `Setting` object. For example:
-    ```python
-    from prefect.settings import PREFECT_HOME
-    PREFECT_HOME.value()
-    ```
-
-    Accessing a setting attribute directly will ignore any `value_callback` mutations.
-    This is not recommended:
-    ```python
-    from prefect.settings import Settings
-    Settings().PREFECT_PROFILES_PATH  # PosixPath('${PREFECT_HOME}/profiles.toml')
-    ```
-    """
+    runner_server_missed_polls_tolerance: int = Field(
+        default=2,
+        description="Number of missed polls before a runner is considered unhealthy by its webserver.",
+    )
 
-    def value_of(self, setting: Setting[T], bypass_callback: bool = False) -> T:
-        """
-        Retrieve a setting's value.
-        """
-        value = getattr(self, setting.name)
-        if setting.value_callback and not bypass_callback:
-            value = setting.value_callback(self, value)
-        return value
+    runner_server_host: str = Field(
+        default="localhost",
+        description="The host address the runner's webserver should bind to.",
+    )
 
-    @field_validator(PREFECT_LOGGING_LEVEL.name, PREFECT_LOGGING_SERVER_LEVEL.name)
-    def check_valid_log_level(cls, value):
-        if isinstance(value, str):
-            value = value.upper()
-        logging._checkLevel(value)
-        return value
+    runner_server_port: int = Field(
+        default=8080,
+        description="The port the runner's webserver should bind to.",
+    )
+
+    runner_server_log_level: str = Field(
+        default="error",
+        description="The log level of the runner's webserver.",
+    )
+
+    runner_server_enable: bool = Field(
+        default=False,
+        description="Whether or not to enable the runner's webserver.",
+    )
+
+    runner_server_log_level: str = Field(
+        default="error",
+        description="The log level of the runner's webserver.",
+    )
+
+    deployment_concurrency_slot_wait_seconds: Annotated[float, Field(ge=0)] = Field(
+        default=30.0,
+        description=(
+            "The number of seconds to wait before retrying when a deployment flow run"
+            " cannot secure a concurrency slot from the server."
+        ),
+    )
```

Comment:
here is the new setting added from https://github.com/PrefectHQ/prefect/pull/15504

Reply:
updated to this in [ffea111](https://github.com/PrefectHQ/prefect/pull/15412/commits/ffea11121e50253408f7ccb5b21a8fc0457903ea)
```python
    deployment_concurrency_slot_wait_seconds: float = Field(
        default=30.0,
        ge=0.0,
        description=(
            "The number of seconds to wait before retrying when a deployment flow run"
            " cannot secure a concurrency slot from the server."
        ),
    )
```